R/fileupload.R

```diff
@@ -20,6 +20,32 @@
 # form upload, i.e. traditional HTTP POST-based file upload) doesn't work with
 # the websockets package's HTTP server at the moment.
 
+#' @details \code{sanitizeFileName} is an effort to safely retain something
```

Comment:
There needs to be one line at the top of the comment block, which gets used as the title of the documentation page.